### PR TITLE
Start working on howgud (#46)

### PR DESCRIPTION
**Pull request by krofna**
This is a draft. Opening to get ideas and feedback

### CODE REVIEW

**Comment by meooow25** [comment]
Ideas for stuff to display:
- How many solved
- Average time taken? (maybe not since gitgud does not incentivize solving fast)
- Histogram of number solved by problem difficulty

**Comment by krofna** [comment]
- number of noguds
- cummulative solved over time line overlaid onto graph (put the different x and y axes top and right)

**Comment by krofna** [comment]
in embed:
- last 5 solved
- current challenge (+current time)

**Comment by krofna** [comment]
image
from test server

File: tle/cogs/codeforces.py [hunk 1]
```diff
@@ -164,7 +164,6 @@ def check(problem):
     @commands.command(brief='Skip challenge')
     @cf_common.user_guard(group='gitgud')
     async def nogud(self, ctx):
-        await cf_common.resolve_handles(ctx, self.converter, ('!' + str(ctx.author),))
```
Comment (meooow25) [inline tle/cogs/codeforces.py]: Why remove this?

File: tle/cogs/graphs.py [hunk 2]
```diff
@@ -489,6 +489,26 @@ def vert_line(x):
         discord_common.set_author_footer(embed, ctx.author)
         await ctx.send(embed=embed, file=discord_file)
 
+    @plot.command(brief='Plot histogram of gudgiting')
+    async def howgud(self, ctx, *members: discord.Member):
+        hist_bins = list(range(-300 + 50, 300 + 50 + 1, 100))
```
Comment (meooow25) [inline tle/cogs/graphs.py]: Would be nice to have some comments about where these constants come from.

**Pull request merged**